app/Policies/EngagePurchaseRequestPolicy.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Models\EngagePurchaseRequest;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class EngagePurchaseRequestPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(User $user): bool
+    {
+        return true;
```

Comment:
You made a permission, shouldn't you use that here?

Reply:
It's a middleware on the route, I don't think it's necessary here https://github.com/RoboJackets/loop/blob/3f9a46ece7710eb81f1fbd502c2181772c53d2e3/routes/api.php#L46